### PR TITLE
[Feature] Provide URLs of Swin Transformer pretrained models

### PR DESCRIPTION
**Pull request by MengzhangLI**
## Motivation

To replace `./pretrain/xxx.pth` in configs of Swin Transformer models with URL of MMSegmentation style pretrained models.

In this way, users of MMSegmentation could directly use these config files to train:

swin

The re-implementing results are below, it seems results are comparable:
<div class="okr-block-clipboard" data-okr="%7B%22okrDelta%22%3A%5B%7B%22lineType%22%3A%22unsupport%22%2C%22lineOptions%22%3A%7B%7D%2C%22lineContent%22%3A%5B%5D%7D%5D%2C%22businessKey%22%3A%22lark-doc%22%7D"></div><div style="white-space: pre;" data-zone-id="0" data-line-index="0">

Method | Backbone | pretrain | [Original mIoU](https://github.com/open-mmlab/mmsegmentation/tree/master/configs/swin) | Re-implement mIoU
-- | -- | -- | -- | --
UperNet | Swin-T | swin_tiny_patch4_window7_224.pth | 44.41 | 44.22
UperNet | Swin-S | swin_small_patch4_window7_224.pth | 47.72 | 47.86
UperNet | Swin-B | swin_base_patch4_window7_224.pth | 47.99 | 48.06
UperNet | Swin-B | swin_base_patch4_window7_224_22k.pth | 50.31 | 49.89
UperNet | Swin-B | swin_base_patch4_window12_384.pth | 48.35 | 48.21
UperNet | Swin-B | swin_base_patch4_window12_384_22k.pth | 50.76 | 50.53

</div>

### CODE REVIEW

**Review by MeowZheng — APPROVED** [review]

File: configs/swin/upernet_swin_base_patch4_window7_512x512_160k_ade20k_pretrain_224x224_1K.py [hunk 1]
```diff
@@ -2,9 +2,13 @@
     './upernet_swin_tiny_patch4_window7_512x512_160k_ade20k_'
     'pretrain_224x224_1K.py'
 ]
+checkpoint_file = 'https://download.openmmlab.com/mmsegmentation/v0.5/pretrain/swin/swin_base_patch4_window7_224_20220317-e9b98025.pth'  # noqa
 model = dict(
-    pretrained='pretrain/swin_base_patch4_window7_224.pth',
+    pretrained=None,
```
Comment (MeowZheng) [inline configs/swin/upernet_swin_base_patch4_window7_512x512_160k_ade20k_pretrain_224x224_1K.py:7]:
```suggestion
```

File: configs/swin/upernet_swin_base_patch4_window7_512x512_160k_ade20k_pretrain_224x224_22K.py [hunk 2]
```diff
@@ -2,4 +2,8 @@
     './upernet_swin_base_patch4_window7_512x512_160k_ade20k_'
     'pretrain_224x224_1K.py'
 ]
-model = dict(pretrained='pretrain/swin_base_patch4_window7_224_22k.pth')
+checkpoint_file = 'https://download.openmmlab.com/mmsegmentation/v0.5/pretrain/swin/swin_base_patch4_window7_224_22k_20220317-4f79f7c0.pth'  # noqa
+model = dict(
+    pretrained=None,
```
Comment (MeowZheng) [inline configs/swin/upernet_swin_base_patch4_window7_512x512_160k_ade20k_pretrain_224x224_22K.py:7]:
```suggestion
```

File: configs/swin/upernet_swin_small_patch4_window7_512x512_160k_ade20k_pretrain_224x224_1K.py [hunk 3]
```diff
@@ -2,8 +2,11 @@
     './upernet_swin_tiny_patch4_window7_512x512_160k_ade20k_'
     'pretrain_224x224_1K.py'
 ]
+checkpoint_file = 'https://download.openmmlab.com/mmsegmentation/v0.5/pretrain/swin/swin_small_patch4_window7_224_20220317-7ba6d6dd.pth'  # noqa
 model = dict(
-    pretrained='pretrain/swin_small_patch4_window7_224.pth',
-    backbone=dict(depths=[2, 2, 18, 2]),
+    pretrained=None,
```
Comment (MeowZheng) [inline configs/swin/upernet_swin_small_patch4_window7_512x512_160k_ade20k_pretrain_224x224_1K.py:7]:
```suggestion
```

File: configs/swin/upernet_swin_tiny_patch4_window7_512x512_160k_ade20k_pretrain_224x224_1K.py [hunk 4]
```diff
@@ -2,9 +2,11 @@
     '../_base_/models/upernet_swin.py', '../_base_/datasets/ade20k.py',
     '../_base_/default_runtime.py', '../_base_/schedules/schedule_160k.py'
 ]
+checkpoint_file = 'https://download.openmmlab.com/mmsegmentation/v0.5/pretrain/swin/swin_tiny_patch4_window7_224_20220317-1cdeb081.pth'  # noqa
 model = dict(
-    pretrained='pretrain/swin_tiny_patch4_window7_224.pth',
+    pretrained=None,
```
Comment (MeowZheng) [inline configs/swin/upernet_swin_tiny_patch4_window7_512x512_160k_ade20k_pretrain_224x224_1K.py:7]:
```suggestion
```

**Comment by MengzhangLI** [comment]
`pretrained=None,` in all 6 config files have been deleted.

**Review by MeowZheng — APPROVED** [review]

**Pull request merged**